project/api/models/account.py

```diff
@@ -259,3 +261,28 @@ def is_full_account(self):
             return True
         else:
             return False
+
+    def get_voted_bills(self):
+        from .activity import Activity
```

Comment:
its to avoid circular dependencies

Reply:
Add that as a comment to the code, so we don't forget.